webpack.config.js

```diff
@@ -3,9 +3,9 @@ const webpack = require('webpack');
 const path = require('path');
 
 module.exports = {
-    entry: "./src/index.js",
+    mode: "production",
     output: {
-        filename: "dist/react-contextmenu.js",
```

Comment:
`dist` is the default output path now, so if we keep it here, we get a file `dist/dist/react-contextmenu.js` :-)

Reply:
Oh, interesting!